packages/entr/path.patch

```diff
@@ -1,26 +0,0 @@
-diff --git a/entr.c b/entr.c
```

Comment:
Why deleting patch?
```
~/Downloads/entr-5.0 $ rg /bin/
NEWS
201: - Set PAGER to an absolute path (/bin/cat)

Makefile.bsd
38:	rm ${DESTDIR}${PREFIX}/bin/entr

entr.c
186:	setenv("PAGER", "/bin/cat", 0);
189:	setenv("SHELL", "/bin/sh", 0);

entr_spec.c
100:	setenv("SHELL", "/bin/Xsh", 1);
679:	ok(strcmp(ctx.exec.file, "/bin/Xsh") == 0);  /* FIXME */
680:	ok(strcmp(ctx.exec.argv[0], "/bin/Xsh") == 0)
```

Reply:
Just saw that this was merged, but `entr_spec.c` wasn't patched